### ISSUE TITLE
ExcludeApp element currently does not work to exclude Teams

### ISSUE THREAD

**Issue by CCDFischer** [MicrosoftDocs/OfficeDocs-DeployOffice#318]
per MO175715 this functionality is currently broken and this page should be updated to state the the steps listed here do not currently work and will not until around April 9th. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7936ca5e-c658-4956-b174-99eaf1eb67d4
* Version Independent ID: 9ecc0b61-a941-f5e1-96c3-d32fae59eacd
* Content: [Deploy Microsoft Teams with Office 365 ProPlus](https://docs.microsoft.com/en-us/DeployOffice/teams-install#feedback)
* Content Source: [DeployOffice/teams-install.md](https://github.com/MicrosoftDocs/OfficeDocs-DeployOffice/blob/live/DeployOffice/teams-install.md)
* Service: **o365-proplus-itpro**
* GitHub Login: @DHB-MSFT
* Microsoft Alias: **danbrown**

**Comment by DHB-MSFT** [comment]
@CCDFischer -- the article was updated last Tuesday to address that functionality issue (see the Important callouts in the article). The sample xml code shows the current workaround. Essentially, there are several places you have to include the ExcludeApp ID="Teams" line in your xml file, depending on what your scenario is.

**Comment by CCDFischer** [comment]
@DHB-MSFT -- that is not how the advisory reads, perhaps it is the advisory that needs updated but as of now it appears as though using the exclude element for teams simply will not work at all until a new ODT is rolled out in April. As we are excluding Teams from our install this is fairly important to us to get and keep working. 
The call out that you reference above is the same as how OneDrive (groove) currently works as well, we need to include the exclude element for our Project and Visio installs as well as our base Office install to prevent OneDrive from being installed with Office.

**Comment by DHB-MSFT** [comment]
@CCDFischer -- thanks for the feedback on the wording of the advisory. I'll see if they can update it to make it clearer. The ExcludeApp element does work for Teams, but only if done in the manner outlined in the article.

**Comment by quinla01** [comment]
I've added the exclusions where mentioned but Teams is still installed during a fresh install. My config content is below. Am I doing something wrong?

<Configuration ID="xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx">
  <Info Description="" />
  <Add OfficeClientEdition="64" Channel="Monthly" SourcePath="\\xxxxxxxxxxxxxxxx" AllowCdnFallback="FALSE" ForceUpgrade="TRUE">
    <Product ID="O365ProPlusRetail">
      <Language ID="en-us" />
      <ExcludeApp ID="Access" />
      <ExcludeApp ID="Groove" />
      <ExcludeApp ID="Lync" />
      <ExcludeApp ID="OneDrive" />
      <ExcludeApp ID="Teams" />
      <ExcludeApp ID="Publisher" />
    </Product>
    <Product ID="VisioProRetail">
      <Language ID="en-us" />
      <ExcludeApp ID="Access" />
      <ExcludeApp ID="Groove" />
      <ExcludeApp ID="Lync" />
      <ExcludeApp ID="OneDrive" />
      <ExcludeApp ID="Teams" />
      <ExcludeApp ID="Publisher" />
    </Product>
  </Add>
  <Property Name="SharedComputerLicensing" Value="1" />
  <Property Name="PinIconsToTaskbar" Value="FALSE" />
  <Property Name="SCLCacheOverride" Value="0" />
  <Updates Enabled="TRUE" />
  <RemoveMSI />
  <AppSettings>
    <Setup Name="Company" Value="xxxxxxxxxxxxxxxxxxx" />
  </AppSettings>
  <Display Level="None" AcceptEULA="TRUE" />
  <Logging Level="Off" />
</Configuration>

**Comment by DHB-MSFT** [comment]
@quinla01 -- to have your xml sample appear here in the comment, I think you need to add ``` as the first line and as the last line of your xml sample.

**Comment by quinla01** [comment]
Thanks for that. I've just adjusted and it seems to have worked.

**Comment by nobenhoffer** [comment]
This still isn't working.  Any eta?

**Comment by KingCuong2** [comment]
I had this issue. Turns out I needed to download the new ODT and extract the new setup.exe. Use that new setup.exe with the config.xml. It should work after that.

**Comment by DHB-MSFT** [comment]
It's always a good idea to make sure you're using the most current version of the Office Deployment Tool, which is available here -- [https://www.microsoft.com/en-us/download/details.aspx?id=49117](https://www.microsoft.com/en-us/download/details.aspx?id=49117).

**Comment by SSpehar** [comment]
I'm having the opposite issue.  Teams is not installing for me as part of even the latest Monthly channel install and I'm using the latest ODT.  I don't have Teams in any exclude mode in my configuration files, though I do have Groove in there as an exclude and that is working.  Ironic that when I actually want Teams to be part of the O365 install it isn't.  :)

**Comment by SSpehar** [comment]
I have to amend my earlier comment.  Apparently I had a "System-Wide Teams Installer" in my Programs and Features list and that was preventing the full O365 install and even the stand alone Teams installer to run.  I removed that, removed the full O365 suite and ran my ODT install again.  I had to reboot before Teams would actually launch and I noticed that the "System-Wide Teams Installer" is back.  Kind of a convoluted way to go about integrating Teams into the O365 install.
